devTools/check_trailing_whitespaces.sh

```diff
@@ -26,6 +26,8 @@ set -Eeuo pipefail
 files_with_trailing_whitespaces=$(
     find . \
         -type f \
+        -not -path "./*.cache" \
+        -not -path "./*.log" \
```

Comment:
Could there be such files deeper in the hierarchy? If so, then this should work better:
```
-not -name \*.cache -not -name \*.log
```